core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -220,11 +223,20 @@ public RemoteLogMetadataManager run() {
         });
     }
 
+    public void endPoint(Optional<EndPoint> endpoint) {
+        this.endpoint = endpoint;
+    }
+
     private void configureRLMM() {
         final Map<String, Object> rlmmProps = new HashMap<>(rlmConfig.remoteLogMetadataManagerProps());
 
         rlmmProps.put(KafkaConfig.BrokerIdProp(), brokerId);
         rlmmProps.put(KafkaConfig.LogDirProp(), logDir);
+        endpoint.ifPresent(e -> {
+            rlmmProps.put("bootstrap.servers", e.host() + ":" + e.port());
```

Comment:
please use the constant `CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG`

Reply:
Yes, but that would add dependency with `client` module with `core`. That's why I use plain string instead

Reply:
ok